src/pages/ListChampions.jsx

```diff
@@ -0,0 +1,69 @@
+import { useState, useEffect } from 'react';
+import ChampionCard from '../components/ChampionCard';
+import { Box, Heading } from '@chakra-ui/react';
+import Background from '../assets/images/BackgroundB2g.jpg';
+import { Skeleton } from '@chakra-ui/react';
+
+function List() {
+  const [champions, setChampions] = useState([]);
+  const [isLoading, setIsLoading] = useState(true);
+
+  useEffect(() => {
+    fetch('//ddragon.leagueoflegends.com/cdn/13.6.1/data/en_US/champion.json')
+      .then(response => response.json())
+      .then(data => {
+        const championsList = Object.values(data.data).map(champion => ({
+          id: champion.id,
+          name: champion.name,
+          image: `//ddragon.leagueoflegends.com/cdn/img/champion/loading/${champion.id}_0.jpg`,
+          tags: champion.tags,
+        }));
+        setChampions(championsList);
+        setIsLoading(false);
+      })
+      .catch(error => {
+        console.error('Error fetching champion data:', error);
+      });
+  }, []);
+
+  return (
+    <Box backgroundImage={`url(${Background})`} backgroundSize='cover' minHeight='100vh' py='4'>
```

Comment:
```suggestion
    <Box backgroundImage={Background} backgroundSize='cover' minHeight='100vh' py='4'>
```
Devrait suffire; + `Background` en minuscule 👀